config/locales/en.yml

```diff
@@ -572,6 +572,19 @@ en:
       applications:
         create:
           notice: Application Registered.
+  errors:
+    contact:
+      html: Feel free to <a href="http://wiki.openstreetmap.org/wiki/Contact" title="Various contact channels explained">contact</a> the OpenStreetMap community if you have found a broken link / bug. Make a note of the exact URL of your request.
```

Comment:
We should avoid including raw html tags in translations - these can be hard for translators to get right, among other things. Instead I think it's better to have each component of the paragraph as separate keys, e.g.
```
errors:
  contact:
    contact_url: http:/...
    contact_url_title: Various...
    contact: contact
    contact_the_community_html: Feel free to %{contact_link} the OpenStreetMap ....
```

... and the appropriate `link_to` when calling the translation.

Reply:
Done.